src/popup/utils/getPopupProps.ts

```diff
@@ -22,8 +21,12 @@ interface IPendingRequest {
 
 type PostMessageReturn = Promise<Partial<IPopupConfig> | null>;
 
-export function buildTx(txType: any) {
-  return TxBuilder.buildTx({ ...txParams[txType] }, txType);
+export function buildTx(txType: keyof typeof Tag) {
+  const params = {
+    ...txParams[txType],
+    tag: Tag[txType] as any,
```

Comment:
doesn't it have a type of sting?

Reply:
Added a todo. It's not the problem with tag. Made txParams as any.